app/src/main/java/com/sopt/now/ui/main/home/MainHomeAdapter.kt

```diff
@@ -32,15 +32,17 @@ class MainHomeAdapter(context: Context) :
         }
     }
 
+    override fun getItemCount(): Int = userList.size
```

Comment:
ListAdapter는 getItemCount() 함수를 선언해줄 필요가 없답니다. 혹시 필요로 하여서 사용하셨을까요?

Reply:
```suggestion
    override fun getItemCount(): Int = currentList.size
```
그리고 만약 필요로 한다면 이런식으로 사용하셔야 합니다!

Reply:
아뇨 리사이클러뷰 어뎁터에서 수정하는 과정에 놓친 거 같습니다 감사해요!